chanzuckerberg/single-cell-data-portal
# test: 5449 shard tests

## Reason for Change

- #5449 

WARNING ⚠️ :
~We had to upgrade to Playwright 1.37.0 to use the merge report function, however, `.only()` is temporarily broken at the moment. See [issue](https://github.com/microsoft/playwright/issues/26492)~ UPDATE: Upgraded to 1.37.1, which fixes the `.only()` issue!

## Changes

1. Upgrade Playwright to the latest (1.37.0)
2. Update e2e test to run with 30 workers
3. NOT running firefox and edge yet, because that's too many workers for GHA to handle. We will revisit testing FF and Edge down the road
4. Add GHA steps to upload and merge sharded Playwright reports

## Testing steps

1. e2e tests should now run with 30 works and roughly ~10 mins
2. You should still see the test report in the GHA summary artifacts section

## Notes for Reviewer
